packages/contracts-bedrock/src/L2/SoulGasToken.sol

```diff
@@ -80,6 +80,16 @@ contract SoulGasToken is ERC20Upgradeable, OwnableUpgradeable {
         require(msg.value == totalValue, "unexpected msg.value");
     }
 
+    /// @notice batchDepositForAll is similar to batchDepositFor, but the value is the same for all accounts.
+    function batchDepositForAll(address[] calldata _accounts, uint256 _value) external payable {
+        require(IS_BACKED_BY_NATIVE, "batchDepositForAll should only be called when IS_BACKED_BY_NATIVE");
```

Comment:
According to the coding style, the assert need to begin with "SGT: "?

Reply:
Done, it's really easy to forget when surrounding examples are not using the style..